genai-perf/tests/test_console_exporter.py

```diff
@@ -165,9 +174,9 @@ def test_embedding_output(self, monkeypatch, capsys) -> None:
         expected_content = (
             "                   NVIDIA GenAI-Perf | Embeddings Metrics                   \n"
             "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┓\n"
-            "┃                    Statistic ┃    avg ┃  min ┃  max ┃  p99 ┃  p90 ┃  p75 ┃\n"
+            "┃ Metric                       ┃    avg ┃  min ┃  max ┃  p99 ┃  p90 ┃  p75 ┃\n"
```

Comment:
@harryskim need your input here.
Is this title better as metric or statistic?

Reply:
It matches with the title more but it is a change i think we need to run by you.